src/Microsoft.Azure.EventHubs/Amqp/AmqpMessageConverter.cs

```diff
@@ -94,17 +94,7 @@ public static AmqpMessage EventDatasToAmqpMessage(IEnumerable<EventData> eventDa
 
         static AmqpMessage EventDataToAmqpMessage(EventData eventData, string partitionKey)
         {
-            AmqpMessage amqpMessage;
-            if (eventData.Body.Array != null)
-            {
-                amqpMessage = AmqpMessage.Create(new Data { Value = eventData.Body });
-            }
-            else
-            {
-                // Empty body
-                amqpMessage = AmqpMessage.Create();
```

Comment:
I am removing this since we never had an eventdata w/o body or we don't allow null body,